groovy-sandbox/README.md

```
@@ -49,16 +49,16 @@ class Test {
 
 ## Contributors
 
-https://github.com/craftercms/craftercms/blob/develop/CONTRIBUTORS.md
+https://github.com/craftersoftware/craftercms/blob/develop/CONTRIBUTORS.md
 
 ## Code of Conduct
```

markdownlint-cli2 (0.23.0)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)